tests/Unit/Helpers/ShiftsTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Engelsystem\Test\Unit\Helpers;
+
+use Engelsystem\Config\Config;
+use Engelsystem\Helpers\Carbon;
+use Engelsystem\Helpers\Shifts;
+use Engelsystem\Test\Unit\TestCase;
+
+class ShiftsTest extends TestCase
+{
+    /**
+     * @covers \Engelsystem\Helpers\Shifts::isNightShift
+     */
+    public function testIsNightShift()
+    {
+        $config = new Config(['night_shifts' => [
+            'enabled'    => false,
+            'start'      => 2,
+            'end'        => 6,
+            'multiplier' => 2,
+        ]]);
+        $this->app->instance('config', $config);
+
+        // At night but disabled
+        $this->assertFalse(Shifts::isNightShift(
+            new Carbon('2042-01-01 04:00'),
+            new Carbon('2042-01-01 05:00')
+        ));
+
+        $config->set('night_shifts', array_merge($config->get('night_shifts'), ['enabled' => true]));
+
+        // Is night shift
```

Comment:
Looks like this are multiple tests here. You can use data providers to easily feed the test function.